.github/workflows/build-lint-test.yml

```
@@ -11,20 +11,25 @@ jobs:
         steps:
             - uses: actions/checkout@v3
 
-            - name: Use Node.js v20
+            - name: Setup pnpm
+              uses: pnpm/action-setup@v2
+              with:
+                  version: '8.8.0'
+
+            - name: Use Node.js '18.12.1'
               uses: actions/setup-node@v2
```

let's use latest action version

```suggestion
              uses: actions/setup-node@v4
```